supervise/main.go

```diff
@@ -45,8 +45,12 @@ MSGID = The message "type". Set to "out" or "err" to specify which stdio
 	address := args[0]
 	componentID := args[1]
 	wd := args[2]
-	program := args[3]
-	arguments := args[4:]
+	timeout, timeoutErr := strconv.Atoi(args[3])
+	if timeoutErr != nil {
+		panic(timeoutErr)
```

Comment:
Use fatalf so that the error can be more easily reported back to the code that started the supervisor.

Reply:
:+1: